build/scripts-3.4/pydeface.py

```diff
@@ -0,0 +1,180 @@
+#!/opt/local/bin/python
+"""
+deface an image using FSL
+USAGE:  deface -i <filename to deface> -o [output filename]
+"""
+
+## Copyright 2011, Russell Poldrack. All rights reserved.
+
+## Redistribution and use in source and binary forms, with or without modification, are
+## permitted provided that the following conditions are met:
+
+##    1. Redistributions of source code must retain the above copyright notice, this list of
+##       conditions and the following disclaimer.
+
+##    2. Redistributions in binary form must reproduce the above copyright notice, this list
+##       of conditions and the following disclaimer in the documentation and/or other materials
+##       provided with the distribution.
+
+## THIS SOFTWARE IS PROVIDED BY RUSSELL POLDRACK ``AS IS'' AND ANY EXPRESS OR IMPLIED
+## WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND
+## FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL RUSSELL POLDRACK OR
+## CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+## CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+## SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON
+## ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING
+## NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF
+## ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+
+import os
+import sys
+import tempfile
+import subprocess
+import inspect
+import argparse
+import nibabel
+from scipy import ndimage as nd
+from nipype.interfaces import fsl
+from pkg_resources import resource_filename, Requirement
+
+
+def run_shell_cmd(cmd,cwd=[]):
+    """
+    Run a command in the shell using Popen
+    """
+    if cwd:
+        process = subprocess.Popen(cmd, shell=True, stdout=subprocess.PIPE, cwd=cwd)
```

Comment:
I think pydeface should generate "moo" sounds by default in honor of this example xD